packages/firestore/test/integration/api/fields.test.ts

```diff
@@ -27,9 +27,10 @@ import {
 } from '../util/helpers';
 import { DEFAULT_SETTINGS } from '../util/settings';
 
-const FieldPath = firebase.firestore!.FieldPath;
-const FieldValue = firebase.firestore!.FieldValue;
-const Timestamp = firebase.firestore!.Timestamp;
+const FieldPath = firebaseExport.FieldPath;
+const FieldValue = firebaseExport.FieldValue;
+const Timestamp = firebaseExport.Timestamp;
+const usesModularApi = firebaseExport.usesFunctionalApi;
```

Comment:
Could we make this consistent throughout?

Reply:
Oops. Done.